src/app/api/recaptcha/route.ts

```
@@ -0,0 +1,34 @@
+    method: "POST",
+    body: formData,
+    headers: {
+      "Content-Type": "application/x-www-form-urlencoded"
+    }
+  })
+
+  if (!res.ok) return NextResponse.json({ success: false })
+
+  const data = await res.json()
+
+  if (data && data.success && data.score > 0.5) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)